_episodes/l2_02_basic_unit_testing.md

```diff
@@ -0,0 +1,285 @@
+---
+title: "Introduction to Unit Testing using GoogleTest"
+teaching: TBC
+exercises: TBC
+questions:
+- "What framework/libraries can be used for unit testing in C++?"
+- "What is GoogleTest?"
+- "How to write unit tests with GoogleTest?"
+objectives:
+- "Understand the basic components of a unit test"
+- "Understand how to write a unit test using GoogleTest"
+- "Understand how to compile and run the tests"
+- "Understand some of the macros available in GoogleTest"
+keypoints:
```

Comment:
I made a comment about this previously, but I think it was maybe lost.

I'd write these points as a series of facts that the student should know, as @dalonsoa did for the previous section: https://github.com/ImperialCollegeLondon/unit_testing_Cpp/blob/25e9189d315ce79a2ab2870fb6611a4b84f2d7b7/_episodes/l1-01-introduction.md?plain=1#L15

That said, it's an extremely minor point and this is totally fine -- especially for a first draft.

Reply:
I think I tried to address your concerns after reading your initial comments. It seems that I did not do well to satisfy you. I am not sure what exactly you want. 

You can suggest a text and I am happy to accept the same.

Reply:
Oh it looks like you've changed it now. Sorry, I must have been looking at an old version.